.stylelintrc.yml

```diff
@@ -0,0 +1,19 @@
+extends:
+  - stylelint-config-standard-scss
+rules:
+  # while we still use node-sass, only legacy rgb() notation is allowed
+  color-function-notation: "legacy"
+  # recommended to turn off descending specificity since we use a lot of nesting:
+  # https://stylelint.io/user-guide/rules/list/no-descending-specificity/
+  no-descending-specificity: null
```

Comment:
It might be nice to have a selector ordering rule. We should look into this further to see how much work it would be and if we should do it in the first place

Reply:
We could handle it as separate task